static/app/views/performance/database/useDenylistedProjects.tsx

```diff
@@ -0,0 +1,36 @@
+import useProjects from 'sentry/utils/useProjects';
+
+interface Options {
+  projectId?: string[];
```

Comment:
l: we might want to rename to `projectIds` to make it clear it is an array.

Reply:
I was torn on this, since most of the app calls it `projectId` despite being plural 😭 keeping as-is for now, but I agree with this complaint!